project/Dependencies.scala

```diff
@@ -77,7 +77,7 @@ object Dependencies {
   def addSbtZincCompile = addSbtModule(sbtZincPath, "zincCompile", zincCompile)
   def addSbtZincCompileCore = addSbtModule(sbtZincPath, "zincCompileCore", zincCompileCore)
 
-  val lmCoursierShaded = "io.get-coursier" %% "lm-coursier-shaded" % "2.1.1"
+  val lmCoursierShaded = "io.get-coursier" %% "lm-coursier-shaded" % "2.1.7"
```

Comment:
Actually this version does not exist since we publish separately from coursier/sbt-coursier.

Also note that sbt is likely not affected by this CVE because we exclude Plexus Archiver.
https://github.com/coursier/sbt-coursier/blob/e2de6e30e32d9eccfab99b38ff4e4d281e00817e/build.sbt#L28-L30

Reply:
This will publish 2.1.2 - https://github.com/coursier/sbt-coursier/actions/runs/6420155362

Reply:
```suggestion
  val lmCoursierShaded = "io.get-coursier" %% "lm-coursier-shaded" % "2.1.2"
```